iml-sfa/src/main.rs

```diff
@@ -68,19 +68,19 @@ async fn main() -> Result<(), ImlSfaError> {
 
         let (new_enclosures, x, new_drives) = future::try_join3(fut1, fut2, fut3).await?;
 
-        tracing::debug!("SfaStorageSystem {:?}", x);
-        tracing::debug!("SfaEnclosures {:?}", new_enclosures);
-        tracing::debug!("SfaDiskDrives {:?}", new_drives);
+        tracing::trace!("SfaStorageSystem {:?}", x);
+        tracing::trace!("SfaEnclosures {:?}", new_enclosures);
+        tracing::trace!("SfaDiskDrives {:?}", new_drives);
```

Comment:
I personally find "trace" useless. If it is worth to exists, it better be something like `trace!("I'm here!")` - i. e. just a mark. When it prints some data, it must be a `debug`.

Reply:
I changed this from debug to trace because it is *very* verbose. In general I won't want to see this being displayed (even when debugging), but every so often I will.